examples/Remarketing/UploadStoreSalesTransactions.cs

```diff
@@ -129,13 +136,13 @@ public override string Description
 
             try
             {
-                // Creates an offline user data job for uploading transactions.
+                // Creates an offline user data job for uploading transactions.   
```

Comment:
Please ensure that there is no trailing whitespace on this line.

Reply:
Removed the whitespace